src/styles/index.scss

```diff
@@ -1,3 +1,211 @@
-body {
+$cardFont: Roboto, Arial, sans-serif;
+$defaultFontWeight: 400;
+$accentColor: #060b35;
+$secondaryColor: #616070;
+$moyoBlue: #00acdc;
+$transitionTime: 300ms;
+
+html {
+  font-size: 12px;
+  font-weight: 500;
```

Comment:
It's better to set the default `font-weight` in the `html` selector to `$defaultFontWeight` variable. This will make your code more consistent and maintainable.